.github/CONTRIBUTING.rst

```diff
@@ -6,7 +6,7 @@ Build Status
 
 - master
 
-  + `Stable buildbots <http://buildbot.python.org/3.x.stable/>`_
+  + `Stable buildbots <http://buildbot.python.org/all/#/masters/>`_
```

Comment:
I think the correct link is either http://buildbot.python.org/all/#/grid?branch=master&tag=stable or http://buildbot.python.org/all/#/builders?tags=%2Bstable&tags=%2B3.x

@zware which one is better?

Reply:
"better" is a pure judgement call, unfortunately.  I prefer the `builders` page, but I believe others prefer the `grid` page.  The grid page does at least attempt to match up builds with commits, so that one may actually be better.

Reply:
@zware I'm going to change the link to the grid page: `http://buildbot.python.org/all/#/grid?branch=master&tag=stable`.